tests/acceptance/30_generic_methods/file_check_exists.cf

```diff
@@ -33,7 +33,7 @@ bundle agent init
   files:
     "${file}"
       create => "true",
-      perms  => mog("000", "root", "root");
+      perms  => mog("000", "root", "0");
```

Comment:
"root" group does not exist on AIX, so we use "0" instead. A style test is added below so this is automatically detected.